backend/src/main/java/se/chalmers/dat265/group1/api/physical/PhysicalsController.java

```diff
@@ -34,12 +41,17 @@ public PhysicalPart updatePhysicalPart(String physicalPartID, PhysicalPart physi
 
     @Override
     public List<PhysicalPrint> getAllPhysicalPrints() {
-        return physicalPrintRepository.getObjects();
+        List<PhysicalPrint> resultList = new LinkedList<PhysicalPrint>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2293)